lib/sablon/template.rb

```diff
@@ -32,6 +33,10 @@ def render(context, properties = {})
             out.write(process(Processor::Document, content, env))
           elsif entry_name == 'word/numbering.xml'
             out.write(process(Processor::Numbering, content))
+          elsif entry_name == 'word/_rels/document.xml.rels'
+            out.write(process(Processor::Image, content, properties))
```

Comment:
For now I'm gonna think on your above suggestions and apply them whenever I can. Thanks!